packages/now-python/now_init.py

```diff
@@ -6,11 +6,11 @@
 from importlib import util
 from http.server import BaseHTTPRequestHandler
 
-# Import relative path https://docs.python.org/3.6/library/importlib.html#importing-a-source-file-directly
+# Import relative path https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly
 __now_spec = util.spec_from_file_location("__NOW_HANDLER_MODULE_NAME", "./__NOW_HANDLER_ENTRYPOINT")
 __now_module = util.module_from_spec(__now_spec)
-__now_spec.loader.exec_module(__now_module)
 sys.modules["__NOW_HANDLER_MODULE_NAME"] = __now_module
+__now_spec.loader.exec_module(__now_module)
 __now_variables = dir(__now_module)
```

Comment:
Thanks!

We don't have unit tests for now-python so please add a [test fixture](https://github.com/zeit/now/tree/master/packages/now-python/test).
Each fixture needs to add a `now.json` with a `probes` property.

Reply:
If I add a test fixture, how to run the test to make sure that my code in that fixture is correct?

Also, any documentation on how to write `probes`? This documentation https://zeit.co/docs/configuration#introduction/configuration-reference doesn't mention it.

Reply:
You can run the tests with `yarn test-integration-once`.

The probes are just a way to make assertions for http requests.

Reply:
Try the following to build everything but only test python:
```
yarn install
yarn build
cd ./packages/now-python
yarn test-integration-once
```

It seems like your example code doesn't work as expected.
I get a response of `{"detail": "Not Found"}` when I run the test.
Can we change the test to read the query string instead of the post body?

Reply:
Thank to the ability to run test on local, I can fix the test case, without quitting "POST" method.

```
  console.log ../../test/lib/deployment/test-deployment.js:32
    testDeployment /home/quan/Works/Forks/now/packages/now-python/test/fixtures/28-pydantic

  console.log ../../test/lib/deployment/now-deploy.js:40
    posting 2 files

  console.log ../../test/lib/deployment/now-deploy.js:57
    id dpl_FzYQ4cHde6kXDrWU8nKVpx8Vi92k

  console.log ../../test/lib/deployment/now-deploy.js:58
    deploymentUrl https://test2020-hgfxkccbw.now.sh

  console.log ../../test/lib/deployment/test-deployment.js:98
    testing {"path":"/","method":"POST","body":{"name":"Nguyễn Hồng Quân"},"status":200}

 PASS  test/test.js (396.883s)
  ✓ should build 01-cowsay (23160ms)
  ✓ should build 02-wsgi-pipenv (17782ms)
  ✓ should build 03-env-vars (16959ms)
  ✓ should build 04-cowsay-pipfile (18037ms)
  ✓ should build 05-cowsay-pipfile-local (15809ms)
  ✓ should build 06-content-type (12771ms)
  ✓ should build 08-django (30452ms)
  ✓ should build 09-url-params (25357ms)
  ✓ should build 10-url-params-http-handler (25139ms)
  ✓ should build 11-asgi (24624ms)
  ✓ should build 12-asgi-function (13220ms)
  ✓ should build 13-wsgi-url-params-encoding (24628ms)
  ✓ should build 14-unicode-handler (13836ms)
  ✓ should build 15-binary-handler (14492ms)
  ✓ should build 20-multivalue-header (34894ms)
  ✓ should build 25-exclude-files (40029ms)
  ✓ should build 27-square-brackets (20819ms)
  ✓ should build 28-pydantic (18631ms)
```